tests/main/interfaces-classic-content-slot/task.yaml

```diff
@@ -0,0 +1,60 @@
+summary: Ensure that the content slots for classic snaps works.
+
+details: |
+    The content-sharing interface interface allows a snap to access contents from
+    other snap. It is possible to have content slots in classic snaps, where
+    confined snaps can connect to.
+
+    A snap which defines the content sharing plug must be shown in the interfaces list.
+    The plug must be autoconnected on install and, as usual, must be able to be
+    reconnected.
+
+# slow in autopkgtest (>4m)
+backends: [-autopkgtest]
```

Comment:
This is not needed because just smoke test suite is executed for autopkgtest backend